rules/src/main/resources/io/github/belgif/rest/guide/validator/rules/oas/Rule-PresenceOfHealthOperation.drl

```diff
@@ -0,0 +1,62 @@
+package io.github.belgif.rest.guide.validator.rules.oas;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import io.github.belgif.rest.guide.validator.core.ViolationReport
+import io.github.belgif.rest.guide.validator.core.parser.Parser.ParserResult
+import io.github.belgif.rest.guide.validator.core.model.PathDefinition;
+import io.github.belgif.rest.guide.validator.core.model.OperationDefinition;
+import io.github.belgif.rest.guide.validator.core.ViolationLevel;
+import org.eclipse.microprofile.openapi.models.responses.APIResponses;
+import org.eclipse.microprofile.openapi.models.Paths;
+import org.eclipse.microprofile.openapi.models.PathItem;
+import org.eclipse.microprofile.openapi.models.Operation;
+global io.github.belgif.rest.guide.validator.core.parser.Parser.ParserResult parserResult;
+global io.github.belgif.rest.guide.validator.core.ViolationReport oas;
+
+function void healthViolation(ViolationReport oas){
+    oas.addViolation("[hlth-res]",
+           		 "Each REST API SHOULD expose a GET /health operation which returns the availability of the API’s functionality for its clients.",
```

Comment:
single static description per fule:

"Each REST API SHOULD expose a GET /health operation which returns the availability of the API’s functionality for its clients." 

msg -  "The HTTP response status SHOULD be either 200 OK when the service is up or partially available, or 503 Service Unavailable when the service is down or out of service."